starters/basic-starter/pages/[...slug].tsx

```diff
@@ -21,24 +22,43 @@ const CONTENT_TYPES = [
   'node--event',
   'node--person',
   'node--place',
+  'taxonomy_term--article_type',
+  'taxonomy_term--categories',
+  'taxonomy_term--event_type',
+  'taxonomy_term--person_type',
+  'taxonomy_term--place_type',
 ];
 
-interface NodePageProps extends LayoutProps {
-  node: DrupalNode;
+interface EntityPageProps extends LayoutProps {
+  node?: DrupalNode;
+  node_list?: DrupalNode[];
+  taxonomy_term?: string;
+  type?: string;
 }
 
-export default function NodePage({ node, menus }: NodePageProps) {
-  if (!node) return null;
-
-  return (
+export default function EntityPage({
+  node,
```

Comment:
Can we call this entity which type would be either `DrupalNode` or `DrupalTaxonomyTerm`?

Reply:
Could you rephrase this? I don't understand